### ISSUE TITLE
make item selection more visually clear

### ISSUE THREAD

**Issue by aspiers** [octobox/octobox#3287]
## 💥 Problem

When an item is selected by hovering the mouse over it, the selection is visually reasonably clear, due to the slightly darker background colour:

![image](https://user-images.githubusercontent.com/100738/227062927-c84ac664-eaa7-4241-8297-3a202995b907.png)

However, if the selection is made via `j` / `k` keyboard shortcuts, there is no change in background colour for the selected item, and the only visual hint of that selection is an extremely thin and hard to spot vertical line:

![image](https://user-images.githubusercontent.com/100738/227063080-77658ab7-fa2a-4767-b3e6-0341e03a74fb.png)

The difficulty of spotting this vertical line is compounded by the fact that if a different item has the mouse over it (as seen here with the item two lines below), that item does not revert back to the default appearance even though it is no longer selected.

I have very good eyesight (when using my glasses or contact lenses!), and even so as someone who prefers to use keyboard over mouse in general, I find it overly difficult to spot this thin vertical line when selecting notifications via keyboard.  It must be a lot harder for someone who is unable to achieve good eyesight.

## Goal
Make the selection visually much clearer.

## Proposed solution
Increasing the width of this vertical line, and perhaps choosing a colour for it which has a much higher contrast to the other colours (i.e. not a grayscale colour) seems like an incredibly quick and easy fix to this problem.

It might also be worth deactivating the mouse hover selection indicators when the keyboard selection deviates from it.

**Comment by aspiers** [comment]
I've made a userstyle which fixes this: https://userstyles.world/style/9261/octobox-io-wider-selection-border

but the actual fix is stupidly easy; just change `1px` to `5px` in

```css
td.current {
    border-left: 1px solid #0366d6 !important;
}
```
